source/contracts/Controller.sol

```diff
@@ -178,4 +179,8 @@ contract Controller is IController {
     function getAugur() public view returns (Augur) {
         return Augur(lookup("Augur"));
     }
+
+    function getTimestamp() public view returns (uint256) {
+        return ITime(lookup("Time")).getTimestamp();
+    }
```

Comment:
We should probably set some ground rules on what decides whether something gets a special getter added to controller.  I think `getTimestamp` may meet those requirements, whatever they are, but I would like to ensure that we are on the same page for what the bar is.

Reply:
It's difficult for me to say exactly why this felt right. In general it seems like small oft used lookups that could be classified at utilites feel like they belong here. Any more concrete proposal?

Reply:
No, I was kind of hoping you had something concrete in your head that we could rally around.  :'(